localized_recurrence/models.py

```diff
@@ -3,6 +3,7 @@
 
 from dateutil.relativedelta import relativedelta
 from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
```

Comment:
https://docs.djangoproject.com/en/1.6/ref/utils/#module-django.utils.encoding

@micahhausler not sure if you've been running into this when upgrading everything to Python3, but what has been your preferred method of handling both __str__ and __unicode__ model methods? This seems to be pretty standard, at least it's what Django itself uses.